en/snap/with-plugins.md

```diff
@@ -56,269 +62,308 @@ Here is the list of Content Management System plugins and extensions that are su
+* Midtrans plugin for WooCommerce [ [GitHub](https://github.com/veritrans/SNAP-Woocommerce) | [Zip](https://github.com/veritrans/SNAP-Woocommerce/archive/master.zip) ]
+
+#### WooCommerce Plugin Installation
+Select **any one** of the installation options given below.
+
+**Simple Installation**
+
+To install Midtrans-WooCommerce plugin, follow the steps given below.
+  1. Login to your WordPress administration panel.
+
+  2. Go to **Plugins** menu.
 
-#### Installation:
+  3. Click **add new**.
 
-Choose **one** from installation options **A or B**:
+  4. Search for **Midtrans-WooCommerce plugin**.
 
-#### A. Simple Installation
-   1. Login to your Wordpress admin panel.
-   2. Go to Plugins menu, click add new. Search for Midtrans-WooCommerce plugin.
-   3. Install and follow on screen instructions.
-   4. Proceed to Configuration Process.
+  5. Click **Install Now** and follow on-screen instructions.
 
-If you are unable to install using this method, please proceed with [manual installation below](#b-manual-installation).
+The plugin is installed successfully. Proceed to [WooCommerce Plugin Configuration](#wooCommerce-plugin-configuration).
+
+If you are unable to install, proceed to Manual Installation.
+
+**Manual Installation**
+
+To install Midtrans-WooCommerce plugin manually, follow the steps given below.
+   1. Download the plugin file from the link given above.
 
-#### B. Manual Installation
-   1. Download the plugin from [Zip](https://github.com/veritrans/SNAP-Woocommerce/archive/master.zip).
    2. Extract the plugin, then rename the modules folder as **midtrans-woocommerce**.
+
    3. Upload the unzipped plugin folder to your WordPress installation's `./wp-content/plugins/` directory.
-   4. Install and activate the plugin from plugins menu within the WordPress admin panel
-   5. Proceed to Configuration Process.
-   
-#### WooCommerce Plugin Configuration Process
-1. Go to **WooCommerce -> Settings -> Payments -> Midtrans** menu, fill the configuration fields.
-    - Fill **Title** with text button that you want to display to customer
-    - Select **Environment**. `Sandbox` for testing transaction and `Production` for real transaction
-    - Fill **Merchant ID, Client Key, and Server key**. You can find [this credential on Midtrans MAP Dashboard](/en/snap/preparation.md).
-    - Other configuration are optional, you may leave it as is.
-
-![WooCommerece Install](./../../asset/image/WooCommerce-install.gif)
 
-#### WooCommerce Plugin Notification Configuration
-Login to your [Midtrans Account](https://account.midtrans.com/login), select your environment (sandbox/production), go to menu `settings` **->** `configuration` and `settings` **->** `Snap Preference` **->** `System Settings`
+   4. On WordPress administration panel, click **Install and activate** the plugin from plugins menu.
+
+The plugin is installed successfully. Proceed to [WooCommerce Plugin Configuration](#wooCommerce-plugin-configuration).
```

Comment:
@rashmikampurath Hey you seem to forget all the basic feedbacks all over again.
```
**Simple Installation**
```
Why does a header got changed to unclickable Bold formatting?

Why is there spacing between numbere lists? Look at your submission render:
> <img width="807" alt="Screen Shot 2020-12-02 at 11 50 10" src="https://user-images.githubusercontent.com/13027142/100829822-dcd9d680-3494-11eb-951b-c72a41415820.png">

Compared to original render:
> <img width="672" alt="Screen Shot 2020-12-02 at 11 49 49" src="https://user-images.githubusercontent.com/13027142/100829862-f11dd380-3494-11eb-9656-30fabff7e0ef.png">

Even the original render is much easier to read visually, it grouped and spaced properly. So you were not introducing improvement here.